orders.go

```diff
@@ -75,6 +75,8 @@ type Order struct {
 	NoteAttributes NoteAttributes
 	// OrderNumber tells you that this is the shop's nth order.
 	OrderNumber int
+	// OrderStatusURL to find the order status
```

Comment:
```suggestion
	// OrderStatusURL is the Thank You page (Shopify doubles it as a order status page)
```
Note that there's 2 URLs for this sorta thing. There's a legit thank you page with `thank_you` in the URL and then this one.
The user is only shown the thank you page once, when it's attempted to be hit again the user is redirected to the OrderStatusURL.
This is why tracking on this page might get complicated. We need that spike I mentioned before on how we can track this.